### PR TITLE
Docker image updates

### PR DESCRIPTION
**Pull request by mobileoverlord**
Trying to accomplish the following
1. Fix an issue with docker running on linux hosts where it would raise an exception that the group id for 1000 does not exist. This is an issue in the docker-entrypoint script. The PR adds the group before trying to add the user to it.
2. Improve docker image layer caching. Since fwup changes more frequently then the reset of the host packages, I am splitting it out in an attempt to see if it can reuse the other layers. I am hoping that this shrinks an 800mb download on every bump to roughly the size of fwup. 
3. Add `libmnl-dev` for building nerves project apps using this image.

### CODE REVIEW

**Comment by michaelkschmidt** [comment]
Each "run" statement creates a unique layer that depends on the layers above it.  If you have something small like fwup that changes alot, move it to the the bottom of the file.  That way it only has to download the "fwup run" layer each time it changes

**Comment by mobileoverlord** [comment]
@michaelkschmidt Ah thanks, I am not sure then if this is going to work. Even though moving it to the bottom would make it smaller. When building the new image with a fwup update it will run the layer above which has a call to `apt-get`. I believe that this causes enough of a ripple each time to mess up the layer cache. Is this true, and if so, is there a way around it?

**Comment by michaelkschmidt** [comment]
Docker actually ignores the fact that `apt-get` might change from run to run.  Unless you explicitly do something to invalidate that layer's cache, it will assume its the same.

For our docker stuff, we always add to the bottom of the file, and then periodically move things higher.  This allows quick updates until we are forced to re-push the world

File: support/docker/nerves_system_br/Dockerfile [hunk 1]
```diff
@@ -50,7 +63,6 @@ RUN dpkg --add-architecture i386 \
     bzip2 \
     bison \
     flex \
-    wget \
```
Comment (fhunleth) [inline support/docker/nerves_system_br/Dockerfile]: Removing `wget` feels like it might break some scripts.

Comment (mobileoverlord) [reply support/docker/nerves_system_br/Dockerfile]: It was moved higher in the script.

**Comment by mobileoverlord** [comment]
Let me fix this up before we release a new system_br

**Comment by mobileoverlord** [comment]
Updated, ready for review

**Pull request merged**